setup.py

```diff
@@ -14,9 +14,10 @@
 """
 
 from setuptools import setup
+import wsgi_kerberos
```

Comment:
You can’t assume the package is importable from within its setup.py, which is run during its installation. setup.py files that single-source the version need to take a different approach. https://packaging.python.org/guides/single-sourcing-package-version/